### PR TITLE
add service worker build and template files

### PR DESCRIPTION
**Pull request by cbessee**
### Description:

This PR adds in the sw-template and sw-build files from the seafood fest map. These files have been updated t0 match the correct mapbox url pattern for chbp and stores have been renamed to reflect chbp ownership.

This addresses #20 

### UI images:

N/A

### Unit Test Approach:

Describe unit tests being added with that PR.

### Test Results:
- [ ] Did you run `npm test` and did the results pass? N/A
- [ ] Did you add unit tests that cover your changes? N/A

Describe other (non-unit) test results here.

### CODE REVIEW

File: src/sw-template.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        cacheName: 'chbp-json'
+      })
+    );
+
+    /* 
+      Custom cache rules - map tiles 
+      Cache map tiles from the external mapbox root url - for only successfull responses.
+    */ 
+    workbox.routing.registerRoute(
+      new RegExp('^https://api.mapbox.com/v4/spatialdev.9w9i8weo,mapbox.mapbox-streets-v8,mapbox.mapbox-terrain-v2/'),
```
Comment (danielduhh) [inline src/sw-template.js:54]: Nice 👍

File: src/sw-template.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    // Cache the Google Fonts stylesheets with a stale-while-revalidate strategy.
+    workbox.routing.registerRoute(
+      /^https:\/\/fonts\.googleapis\.com/,
+      new workbox.strategies.StaleWhileRevalidate({
+        cacheName: 'google-fonts-stylesheets'
+      })
+    );
+
+    // Cache the underlying font files with a cache-first strategy for 1 year.
+    workbox.routing.registerRoute(
```
Comment (danielduhh) [inline src/sw-template.js:74]: Can we also cash this font we're brining it from some outside source?
```
@font-face {font-family: 'Gobold Bold Italic';
    src: url('https://db.onlinewebfonts.com/t/dbd939c1aa46674fe3c9bedb1a7d2730.eot'); /* IE9*/
    src: url('https://db.onlinewebfonts.com/t/dbd939c1aa46674fe3c9bedb1a7d2730.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
    url('https://db.onlinewebfonts.com/t/dbd939c1aa46674fe3c9bedb1a7d2730.woff2') format('woff2'), /* chrome firefox */
    url('https://db.onlinewebfonts.com/t/dbd939c1aa46674fe3c9bedb1a7d2730.woff') format('woff'), /* chrome firefox */
    url('https://db.onlinewebfonts.com/t/dbd939c1aa46674fe3c9bedb1a7d2730.ttf') format('truetype'), /* chrome firefox opera Safari, Android, iOS 4.2+*/
    url('https://db.onlinewebfonts.com/t/dbd939c1aa46674fe3c9bedb1a7d2730.svg#Gobold Bold Italic') format('svg'); /* iOS 4.1- */
}
```

**Review by danielduhh — COMMENTED** [review]
Looks good- just a request to cache some other font files

File: src/sw-template.js (hunk 2 above)
Comment (cbessee) [reply src/sw-template.js:74]: Added in most recent commit

Comment (danielduhh) [reply src/sw-template.js:74]: 🚗 💨

**Review by danielduhh — APPROVED** [review]

**Pull request merged**